plane/source/docs/apms-failsafe-function.rst

```diff
@@ -7,111 +7,85 @@ Plane Failsafe Function
+#. Detects a RC Failsafe condition and then initiating a defined response, such as returning to home. Detection of an RC Failsafe is either a complete loss, or corruption, of RC signals, or the receiver sets a FS bit in its data stream for those protocols supporting it (SBUS, etc.), or that the throttle channel PWM value falls below a certain point set by :ref:`THR_FS_VALUE<THR_FS_VALUE>`. This RC failsafe must be enabled by enabled by by setting :ref:`THR_FAILSAFE<THR_FAILSAFE>` = 1.
 #. Optionally, detect loss of telemetry (GCS Failsafe) and take an programmable action, such as switching to return to launch (RTL) mode.
 #. Detect loss of GPS for more than 20 seconds and switch into Dead Reckoning mode until GPS signal is regained. See https://youtu.be/0VMx2u8MlUU for a demo.
-#. Optionally, detect low battery conditions (voltage/remaining capacity) and initiate a programmable response, such as returning to home. ArduPilot supports this on multiple batteries.
+#. Optionally, detect low battery conditions (low voltage/remaining capacity) and initiate a programmable response, such as returning to home. ArduPilot supports this on multiple batteries.
 
 Here's what the failsafe **will not do**:
 
-#. Detect if one or more individual RC channel has failed or become disconnected
-#. Detect if you're flying too far away or are about to hit the ground
-#. Detect autopilot hardware failures, such as low-power brownouts or in-air reboots
-#. Detect if the Plane software is not operating correctly
-#. Detect other problems with the aircraft, such as motor failures 
-#. Otherwise stop you from making setup or flight mistakes
+- Detect if one or more individual RC channel has failed or become disconnected
+- Detect if you're flying too far away or are about to hit the ground
+- Detect autopilot hardware failures, such as low-power brownouts or in-air reboots
+- Detect if the Plane software is not operating correctly
+- Detect other problems with the aircraft, such as motor failures 
+- Otherwise stop you from making setup or flight mistakes
 
 
 .. note:: See :ref:`advanced-failsafe-configuration` for extended failsafe configurations.
 
 
-Plane Failsafe Documentation
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+RC Failsafe
+===========
 
 .. _apms-failsafe-function_throttle_failsafe:
 
+Radio Signal Failure
+~~~~~~~~~~~~~~~~~~~~
+
+If the received signal is lost or the control information corrupted, or the receiver sets its "failsafe bit" in protocols which have this (receiver protocols which have this bit are Sbus, SXRL, SXRL2, SumD, FPort, FPort2, and iBUS.), then an RC Failsafe condition occurs and the actions described in the :ref:`RC Failsafe Actions<fs_actions>` section below will be taken, if the :ref:`THR_FAILSAFE<THR_FAILSAFE>` parameter is 1.
```

Comment:
I think CRSF have failsafe bit also? Also same as above with note of RC_OPTIONS bit 2

Reply:
maybe just omit the list so that it is not something to be updated later when new protocols added?

Reply:
thanks...changed